src/main/java/models/Auction.java

```diff
@@ -98,7 +104,12 @@ public void closeAuction() {
                 totalSoldQuantity += bid.getQuantity();
                 remainingQuantity -= bid.getQuantity();
             } else {
-                winningBids.add(new Bid(bid.getBidder(), symbol, id, bid.getPrice(), remainingQuantity, bid.getSubmissionTime()));
+                try {
```

Comment:
this stands out from the rest of the "dumb" model as something that should potentially be in service layer (or auction manager) instead of model